test/integration/auth.spec.ts

```diff
@@ -1154,6 +1154,55 @@ describe('admin.auth', () => {
     });
   });
 
+  describe('Project config management operations', () => {
+    before(function() {
+      if (authEmulatorHost) {
+        this.skip(); // getConfig is not supported in Auth Emulator
+      }
+    });
+    const projectOption1: UpdateProjectConfigRequest = {
+      recaptchaConfig: {
+        emailPasswordEnforcementState:  'AUDIT',
+        managedRules: [{ endScore: 0.1, action: 'BLOCK' }],
+      }
+    }
+    const projectOption2: UpdateProjectConfigRequest = {
+      recaptchaConfig: {
+        emailPasswordEnforcementState:  'OFF',
+      }
+    }
+    const expectedProjectConfig1: any = {
+      recaptchaConfig: {
+        emailPasswordEnforcementState:  'AUDIT',
+        managedRules: [{ endScore: 0.1, action: 'BLOCK' }],
+      }
+    }
+    const expectedProjectConfig2: any = {
+      recaptchaConfig: {
+        emailPasswordEnforcementState:  'OFF',
+        managedRules: [{ endScore: 0.1, action: 'BLOCK' }],
+      }
+    }
+    it('updateProjectConfig() should resolve with the updated project config', () => {
+      return getAuth().projectConfigManager().updateProjectConfig(projectOption1)
+        .then((actualProjectConfig) => {
+          expect(actualProjectConfig.toJSON()).to.deep.equal(expectedProjectConfig1);
+          return getAuth().projectConfigManager().updateProjectConfig(projectOption2);
+        })
+        .then((actualProjectConfig) => {
+          expect(actualProjectConfig.toJSON()).to.deep.equal(expectedProjectConfig2);
+        });
+    });
+    it('getProjectConfig() should resolve with expected project config', () => {
+      return getAuth().projectConfigManager().getProjectConfig()
+        .then((actualConfig) => {
+          const actualConfigObj = actualConfig.toJSON();
+          expect(actualConfigObj).to.deep.equal(expectedProjectConfig2);
```

Comment:
Can we guarantee that this test always gets executed after the previous one? Otherwise, you might want to update the config in a `before` block.

Reply:
Mocha runs tests in serial unless indicates otherwise by using `--parallel` parameter.

Reply:
Great! Thanks.